src/options/set.rs

```diff
@@ -195,7 +195,6 @@ pub fn set_options(
         opt.side_by_side = false;
         opt.file_decoration_style = "none".to_string();
```

Comment:
This is required.
file_decoration_style will break structures anyway. We should prevent any of them.